src/Uuid/Uuids.php

```diff
@@ -79,6 +79,11 @@ public static function each(Closure $callback, string $type = 'all'): void
 		// }
 	}
 
+	public static function enabled(): bool
+	{
+		return App::instance()->option('content.uuid') !== false;
```

Comment:
For strict checking of the option value:

```suggestion
		return App::instance()->option('content.uuid', true) === true;
```

Reply:
Since the default is activated, I think it makes more sense to check explicitly for false. And all other values still don't deactivate UUIDs, only false does.